remote/setup_workspace.py

```diff
@@ -37,19 +39,45 @@ def replace_config(path: pathlib.Path, key: str, value: str) -> None:
     path.write_text(file_content)
 
 
+def replace_config_repositories(key: str, value: str) -> None:
```

Comment:
```suggestion
def inject_t4c_connection_details(name: str, value: str) -> None:
```

I'd choose another name here (describing what the function does). It's not really clear from `key` what it is about. And I'd replace `value`, see the comment below.